sips/sip-028/sip-028-sbtc_peg.md

```diff
@@ -0,0 +1,217 @@
+# SIP-028: sBTC Signer Criteria
+
+**SIP Number:** 028  
+**Title:** Signer Criteria for sBTC, A Decentralized and Programmable Asset Backed 1:1 with BTC  
+**Consideration:** Technical, Economics  
+**Type:** Standard  
+**Status:** Draft  
+**Authors:**  
+- Andre Serrano ([andre@bitcoinl2labs.com](mailto:andre@bitcoinl2labs.com))  
+- Ashton Stephens ([ashton@trustmachines.co](mailto:ashton@trustmachines.co))  
+- Joey Yandle ([joey@trustmachines.co](mailto:joey@trustmachines.co))  
+- Mårten Blankfors ([marten@trustmachines.co](mailto:marten@trustmachines.co))  
+- Jesus Najera ([jesus@stratalabs.xyz](mailto:jesus@stratalabs.xyz))  
+- Jude Nelson ([jude@stacks.org](mailto:jude@stacks.org))  
+- Friedger Müffke ([friedger@ryder.id](mailto:friedger@ryder.id))  
+- Tycho Onnasch ([tycho@zestprotocol.com](mailto:tycho@zestprotocol.com))  
+- Daniel Jordon ([daniel@trustmachines.co](mailto:daniel@trustmachines.co))
+
+## Abstract
+
+This SIP takes the position that Stacks can play a key role in offering a rich programming environment for Bitcoin with low-latency transactions. This would be achieved with a new wrapped Bitcoin asset, called sBTC, which would be implemented on Stacks 3.0 and later as a SIP-010 token. Stacks today offers a smart contract runtime for Stacks-hosted assets, and the forthcoming Stacks [3.0 release](https://github.com/stacksgov/sips/blob/feat/sip-021-nakamoto/sips/sip-021/sip-021-nakamoto.md) provides lower transaction latency than Bitcoin for Stacks transactions. By providing a robust BTC-wrapping mechanism based on [threshold signatures](https://eprint.iacr.org/2020/852.pdf), users would be able to lock their real BTC on the Bitcoin chain, instantiate an equal amount of sBTC tokens on Stacks, use these sBTC tokens on Stacks, and eventually redeem them for real BTC at 1:1 parity, minus the cost of the relevant blockchain transaction fees.
```

Comment:
Nit: Shorten `This SIP takes the position that Stacks can play a key role in offering a rich programming environment for Bitcoin with low-latency transactions. This would be achieved with a new wrapped Bitcoin asset...` to `This SIP proposes a new wrapped Bitcoin asset...`.

Reply:
d5b1ac0750e2d5c5690f87a8f5473ad755019545